src/AI/SpenderAI.java

```diff
@@ -177,19 +178,34 @@ else if( unitQueue.isEmpty() )
           // Find the possible destinations.
           ArrayList<XYCoord> destinations = Utils.findPossibleDestinations(unit, gameMap);
 
-          Utils.sortLocationsByDistance(new XYCoord(unit.x, unit.y), unownedProperties);
           if( !unownedProperties.isEmpty() ) // Sanity check - it shouldn't be, unless this function is called after we win.
           {
             log(String.format("  Seeking a property to send %s after", unit.toStringWithLocation()));
             int index = 0;
             XYCoord goal = null;
             Path path = null;
             boolean validTarget = false;
+            ArrayList<XYCoord> validTargets = new ArrayList<>();
 
+            if( unit.model.chassis == ChassisEnum.TROOP ) // Technically a hack, but we don't have vehicles that can capture... yet.
```

Comment:
A better/more futureproof solution would be to see if `Unit.model.possibleActions` contains`ActionType.CAPTURE`.

Reply:
It would be, if possibleActions was not a basic array.
I tried that first and realized I would have to write a loop.

Reply:
Are you saying there's some reason that relying on a basic array may cause problems? Or do you just not want to write a few extra lines of code?

Reply:
I'm saying I'm lazy.

Reply:
Ah. Well then, as a lazy person, you should recognize that fixing it now will be less effort than running into a problem with it later, and having to re-figure out what the issue is before doing what you should have done now. It's really less effort to do it right the first time. Plus, then future you won't hate past you. Always try to do future you a solid.